k9mail/build.gradle

```diff
@@ -245,73 +245,58 @@ android {
         }
     }
 
-    flavorDimensions "type", "store"
+    flavorDimensions = ["type"]
 
     productFlavors {
 
         enterprise {
-            buildConfigField "boolean", "IS_ENTERPRISE", "true"
+            buildConfigField "boolean", "IS_OFFICIAL", "true"
```

Comment:
this buildConfigField has been renamed to IS_OFFICIAL and it is set to true in all flavours, is there any case where it is false?

Reply:
yes, it is only the case of "dev" flavor, which is not official and only admits debug build (no releases). We keep dev flavor just because it has more settings etc that sometimes can come in handy. By default the dev flavor is disabled, not even debug build allowed, but it can be enabled with a boolean flag in customConfig.gradle file.